config.toml

```diff
@@ -0,0 +1,5 @@
+# configuration for automated builds
+
+latest = "v1.20"
+fullversion = "v1.20.0"
```

Comment:
When will we use the `fullversion` vs the `version`

Reply:
I am not sure if it is needed.

Reply:
Oh. The full version could be used in the installation string. Right now a '0' is appended.